Source/Extensions/Blazorise.Charts.DataLabels/ChartDataLabels.razor.cs

```diff
@@ -1,16 +1,87 @@
 ﻿#region Using directives
 using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.Threading.Tasks;
+using Blazorise.Charts.Trendline;
 using Blazorise.Extensions;
 using Microsoft.AspNetCore.Components;
 using Microsoft.JSInterop;
 #endregion
 
 namespace Blazorise.Charts.DataLabels
 {
-    public partial class ChartDataLabels<TItem> : BaseComponent
+    /// <summary>
+    /// Highly customizable Chart.js plugin that displays labels on data for any type of charts.
+    /// </summary>
+    /// <typeparam name="TItem">Generic dataset value type.</typeparam>
+    public partial class ChartDataLabels<TItem> : BaseComponent, IAsyncDisposable
     {
+        #region Methods
+
+        /// <inheritdoc/>
+        protected override Task OnInitializedAsync()
+        {
+            if ( JSModule == null )
```

Comment:
Weren't we starting to use `is` `is not` for the null checking? Or old habbits die hard?? :)
Also check might be redudant.

Reply:
What would happen if an actual existing module was there for some weird reason? Then it wouldn't run `SetDataLabels`? Intended?

Reply:
> Or old habbits die hard?? :)

Yes, this :D

---

> What would happen if an actual existing module was there for some weird reason? Then it wouldn't run SetDataLabels? Intended?

I don't see a reason for it to be anything else but null. We make this pattern everywhere.